src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -665,13 +672,13 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                         Error(diagnostics, ErrorCode.ERR_AnonymousTypeDuplicatePropertyName, argumentSyntax.NameColon.Name);
```

Comment:
@jcouv You are missing the point. I am talking about errors reported for the same element. Its name can be both invalid and duplicate. I would expect us to report only one error for this element in this case.